Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM python:3.10
+FROM python:3.11
 ARG UID=1001
 ARG GID=1001
 RUN apt update -y && apt upgrade -y \
-    && apt install libfreetype6-dev libxft-dev fortune -y
+    && apt install libfreetype6-dev libxft-dev fortunes -y
```

Comment:
This includes fortune right? What's the difference between fortune and fortunes?

Reply:
I think the underlying distro changed their packaging, `fortune` is now just the program while `fortunes` is the data files